app/controllers/reviewable_claimed_topics_controller.rb

```diff
@@ -38,7 +38,7 @@ def destroy
   private
 
   def notify_users(topic, claimed_by)
-    user_ids = User.staff.pluck(:id)
+    user_ids = User.human_users.staff.pluck(:id)
```

Comment:
I'm wondering why we don't just restrict the `MessageBus` message to the `staff` group instead of having to pluck all the user ids here.